zebra-chain/src/transparent/opcodes.rs

```diff
@@ -0,0 +1,13 @@
+//! Zebra script opcodes.
+
+/// Supported opcodes
+pub enum OpCode {
```

Comment:
This is part of the spec, so we need to quote where it comes from.

```suggestion
/// Supported opcodes
///
/// <https://github.com/zcash/zcash/blob/8b16094f6672d8268ff25b2d7bddd6a6207873f7/src/script/script.h#L39>
pub enum OpCode {
```